doc/FAQ.rst

```diff
@@ -144,8 +169,17 @@ web. Useful tricks to reduce file size include:
   ``webm``, ``mp4`` or animated ``gif``, while being aware that those
   formats may introduce visible artifacts.
 
+It is also possible to generate web pages that do not actually include
+all of the data shown, by specifying a `DynamicMap
+<https://github.com/ioam/holoviews/blob/babac9e2916384e03cd189865d5ca59a6b490708/holoviews/core/spaces.py#L332>`_
+rather than a HoloMap.  The DynamicMap will request data only as
+needed, and so requires a Python server to be running alongside the
+viewable web page.  Such pages are more difficult to share by email or
+on web sites, but much more feasible for large datasets.
+
+  
 **Q: How do I create a Layout or Overlay object from an arbitrary list?**
 
-You can supply the list of elements directly to the ``Layout`` and
+You can supply a list of ``elements`` directly to the ``Layout`` and
```

Comment:
Ah ok, I didn't get that. The constructor accepts a list of elements but that argument isn't most likely called ``data`` because there are multiple ways it can be used.